frontend/src/components/analysis/result/AnalysisInfoCard.jsx

```diff
@@ -0,0 +1,151 @@
+            {!isEditing && <h3 className="">{analysisName}</h3>}
+            {isEditing && (
+              <Input
+                id="edit_analysis-input"
+                name="textArea"
+                type="textarea"
+                onChange={(event) => {
+                  setAnalysisName(event.target.value);
+                }}
+                value={analysisName}
+                style={{
+                  maxWidth: "600px",
+                  maxHeight: "20px",
+                  overflowX: "scroll",
+                }}
+                className="me-2 bg-dark"
+              />
+            )}
+            {!isEditing && (
+              <IconButton
+                id="edit-analysis-name"
+                Icon={MdEdit}
+                color=""
+                className="text-secondary"
+                onClick={() => setIsEditing(true)}
+                title="Edit name"
+                titlePlacement="top"
+              />
+            )}
+            {isEditing && (
+              <IconButton
+                id="view-analysis-name"
+                Icon={MdRemoveRedEye}
+                color=""
+                className="text-secondary px-1"
+                onClick={editAnalysisName}
+                title="View name"
+                titlePlacement="top"
+              />
+            )}
+          </Col>
+          <Col sm={12} md={1} className="d-flex justify-content-end">
+            <Button
+              className="bg-darker border-0"
+              onClick={() => setIsOpen(!isOpen)}
+              id="AnalysisInfoCardDropDown"
+            >
+              <ArrowToggleIcon isExpanded={isOpen} />
+            </Button>
+            <UncontrolledTooltip
+              placement="left"
+              target="AnalysisInfoCardDropDown"
+            >
+              Toggle Analysis Metadata
+            </UncontrolledTooltip>
+          </Col>
+        </Row>
+      </ContentSection>
+      <Collapse isOpen={isOpen} id="AnalysisInfoCardCollapse">
+        <ContentSection className="border-top-0 bg-body ps-0 pe-1 py-1">
+          <ListGroup
+            horizontal
+            className="align-items-start flex-wrap flex-lg-nowrap"
+          >
+            {[
+              ["Status", <StatusTag status={analysis.status} />],
+              ["TLP", <TLPTag value={analysis.tlp} />],
+              [
+                "Tags",
+                analysis.tags.map(
+                  (tag) =>
+                    tag !== null && (
+                      <JobTag key={tag.label} tag={tag} className="me-2" />
+                    ),
+                ),
+              ],
+              ["User", analysis.owner],
+              [
+                "Start Time",
+                <DateHoverable
+                  id={`overview-start_time__${analysis.id}`}
+                  value={analysis.start_time}
+                  format="hh:mm:ss a MMM do, yyyy"
+                />,
+              ],
+            ].map(([key, value]) => (
```

Comment:
I suggest to use differnt names instead of key, value. For example jobFieldName, jobFieldValue or something like that